app/javascript/src/Users/permissions.jsx

```diff
@@ -228,7 +228,8 @@ export const Form = { // eslint-disable-line no-unused-vars
       settings: 'Settings',
       partners: 'Developer Accounts -- Applications',
       monitoring: 'Analytics',
-      plans: 'Integration & Application Plans'
+      plans: 'Integration & Application Plans',
+      policies_registry: 'Policies Registry'
```

Comment:
Use cammelCase please

Reply:
It's not gonna be possible, these come from Rails.

Reply:
features are filtered by the permissions defined here: https://github.com/3scale/porta/blob/053897a3f3b696e91957e6daf42d99d44eea2f82/app/models/admin_section.rb#L3